src/app/lib/config/services/magyarul.ts

```diff
@@ -6,311 +6,392 @@ import { DefaultServiceConfig } from '../../../models/types/serviceConfig';
 
 export const service: DefaultServiceConfig = {
   default: {
-    lang: 'en-GB',
-    articleAuthor: `https://www.facebook.com/bbcnews`,
-    articleTimestampPrefix: 'Updated',
+    lang: 'hu',
+    articleAuthor: 'https://www.facebook.com/bbcnews',
+    articleTimestampPrefix: 'Frissítve',
     articleTimestampSuffix: '',
-    atiAnalyticsAppName: 'news',
+    atiAnalyticsAppName: 'news-magyarul',
     atiAnalyticsProducerId: '134',
     atiAnalyticsProducerName: 'HUNGARIAN',
-    useReverb: true,
```

Comment:
Can we reinstate useReverb (apologies the config was copied from news where this value is not set).